pkg/cli/admin/release/info.go

```diff
@@ -775,6 +775,20 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 				errs = append(errs, err)
 				return true, nil
 			}
+			for _, tag := range is.Spec.Tags {
+				// If ForcePrefix true, use user-provided image rather than it's mirrored source
+				// imagereference.Parse returns the digest ID of each component in the release image-reference.
+				// If can't get digest ID, skip this tag, this happens when user has built a payload by
+				// replacing component images in the release with a new image
```

Comment:
Is this "user builds a release replacing a component image with a by-tag pullspec"?  Can we forbid that?  I would hope users replacing component images would use by-digest pullspecs, and expect the new release image to include digests for those images too.

Reply:
When I create a release payload w/ a substituted image I pass something like `cli=quay.io/sallyom/cli:test` I think that's what most devs would do? That's the case I added that skip for.  :shrug: